Task 5/Array Helpers.js

```diff
@@ -0,0 +1,26 @@
+Array.prototype.square = function(){ return this.map( value => value*value ) };
+Array.prototype.cube = function(){ return this.map( value => value*value*value ) };
+Array.prototype.sum = function(){ 
+  let sum = 0;
+  this.forEach( value => sum += value );
+  return sum;
```

Comment:
This approach works. You can also achieve the same result bu using the `reduce` array helper method.

```
Array.prototype.sum = function() {
        return this.reduce((a,b) => a+b);
} 
```

Array.reduce MDN Reference
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/reduce